### PR TITLE
Fix pylint deprecation warnings

### PR DESCRIPTION
**Pull request by ulliholtgrave**
### Short description

I was kinda annoyed by the `distutils` deprecation warning and solved them for our project by following the PEP 632 recommendations (see [https://peps.python.org/pep-0632/](https://peps.python.org/pep-0632/)).

### Proposed changes

- Add a general utils file to the top level of our project if our modules are needing more shared methods.

### Resolved Issues
Fixes: #2122 

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

File: integreat_cms/default_utils.py
Comment (timobrembeck) [inline integreat_cms/default_utils.py:1]: Just a matter of preference, but I think plain `utils.py` is sufficient as filename, no need for the `default_` prefix.
We also used this convention in the other Django apps, so we either have a single `utils.py` file or a `utils` directory containing multiple utils with prefixes...

File: integreat_cms/release_notes/current/unreleased/2406.yml
Comment (timobrembeck) [inline integreat_cms/release_notes/current/unreleased/2406.yml:1]: Implementation details are irrelevant for the end users and other teams, so I would suggest to remove this release note to stay consistent with our previous guidelines.

**Review by timobrembeck — CHANGES_REQUESTED** [review]

File: integreat_cms/utils.py
Comment (timobrembeck) [inline integreat_cms/utils.py:1]: Sorry, I think I have expressed myself a little misleadingly... I think the location of the file inside the `core` directory was better than in the root directory, I just suggested it to rename it. So now the name is better but I'd suggest to move it back into the core dir :see_no_evil:

File: integreat_cms/api/v3/locations.py [hunk 1]
```diff
@@ -12,6 +10,7 @@
 from ...cms.constants import status
 from ...cms.models import POICategoryTranslation
 from ...cms.models.pois.poi import get_default_opening_hours
+from ...default_utils import strtobool
```
Comment (timobrembeck) [inline integreat_cms/api/v3/locations.py:13]:
```suggestion
from ...core.utils import strtobool
```

File: integreat_cms/core/settings.py [hunk 2]
```diff
@@ -10,13 +10,13 @@
...
 import os
-from distutils.util import strtobool
 from urllib.parse import urlparse
 
 from django.core.exceptions import ImproperlyConfigured
 from django.utils.translation import gettext_lazy as _
 
 from ..nominatim_api.utils import BoundingBox
+from ..utils import strtobool
 from .logging_formatter import ColorFormatter, RequestFormatter
```
Comment (timobrembeck) [inline integreat_cms/core/settings.py:20]:
```suggestion
from .logging_formatter import ColorFormatter, RequestFormatter
from .utils import strtobool
```

**Review by timobrembeck — CHANGES_REQUESTED** [review]

**Review by timobrembeck — APPROVED** [review]

**Review by PeterNerlich — APPROVED** [review]
LGTM!

**Pull request merged**